src/query/config/src/inner.rs

```diff
@@ -382,3 +342,49 @@ impl Debug for MetaConfig {
             .finish()
     }
 }
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum CatalogConfig {
+    Hive(CatalogHiveConfig),
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum ThriftProtocol {
+    Binary,
+    // Compact,
```

Comment:
Remove this line if we are not supporting this. We can add a doc comments for `ThriftProtocol` like:

```
/// TODO: add compact protocol support if needed
pub enum ThriftProtocol {
```